core/server/data/schema/schema.js

```diff
@@ -296,6 +296,11 @@ module.exports = {
         id: {type: 'string', maxlength: 24, nullable: false, primary: true},
         event: {type: 'string', maxlength: 50, nullable: false, validations: {isLowercase: true}},
         target_url: {type: 'string', maxlength: 2000, nullable: false},
+        name: {type: 'string', maxlength: 191, nullable: true},
+        secret: {type: 'string', maxlength: 191, nullable: true},
+        api_version: {type: 'string', maxlength: 191, nullable: false, defaultTo: 'v2'},
```

Comment:
These are the groups we use, [see](https://github.com/TryGhost/Ghost/pull/7932).

191 for an api_version feels a little too high?
I think 50 is enough?

Reply:
```suggestion
        api_version: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'v2'},
```